appinventor/appengine/src/com/google/appinventor/client/style/neo/Ode.ui.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2009-2011 Google, All Rights reserved -->
+<!-- Copyright 2011-2023 MIT, All rights reserved -->
+<!-- Released under the Apache License, Version 2.0 -->
+<!-- http://www.apache.org/licenses/LICENSE-2.0 -->
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+             xmlns:g="urn:import:com.google.gwt.user.client.ui"
+             xmlns:ed="urn:import:com.google.appinventor.client.editor.youngandroid"
+             xmlns:ya="urn:import:com.google.appinventor.client.editor.youngandroid.actions"
+             xmlns:neo="urn:import:com.google.appinventor.client.style.neo"
+             xmlns:box="urn:import:com.google.appinventor.client.boxes"
+             xmlns:widgets="urn:import:com.google.appinventor.client.widgets"
+             ui:generatedFormat="com.google.gwt.i18n.server.PropertyCatalogFactory"
+             ui:generatedKeys="com.google.gwt.i18n.server.keygen.MethodNameKeyGenerator"
+             ui:generateLocales="default">
+  <ui:with field="style" />
+  <ui:with field="messages" type="com.google.appinventor.client.OdeMessages" />
+  <g:FlowPanel styleName="ode-Root">
+    <neo:TopPanelNeo ui:field="topPanel" />
+    <g:FlowPanel styleName="ode-TutorialWrapper" ui:field="overDeckPanel">
+      <ed:TutorialPanel ui:field="tutorialPanel" width="100%" height="100%" visible="false" />
+      <g:DeckPanel ui:field="deckPanel" animationEnabled="true" styleName="ode-DeckPanel">
+        <g:FlowPanel styleName="ode-ProjectListView" width="100%" >
```

Comment:
Consider moving width/height settings to CSS.

Reply:
> Consider moving width/height settings to CSS.

To my surprise, I can't find a definition for ode-ProjectListView anywhere. It's used in both Ode.ui.xml files, and I'm sure it contained something at one time. Both layouts display fine without reference to this class. I'm removing it.